src/Microsoft.Health.Dicom.Core/Messages/ExtendedQueryTag/GetExtendedQueryTagErrorsResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+
+namespace Microsoft.Health.Dicom.Core.Messages.ExtendedQueryTag
+{
+    public class GetExtendedQueryTagErrorsResponse
+    {
+        public GetExtendedQueryTagErrorsResponse(
+            string tagPath,
+            IEnumerable<ExtendedQueryTagError> extendedQueryTagErrors
+            )
+        {
+            Path = tagPath;
+            ExtendedQueryTagErrors = extendedQueryTagErrors;
+        }
+
+        public string Path { get; set; }
```

Comment:
Should we echo the tag path even though presumably the user knows this, as the path includes it?

Reply:
I think that is what we do for GetExtendedQueryTag